src/frontend/screens/Observation/FieldDetails.tsx

```diff
@@ -1,27 +1,28 @@
 import * as React from 'react';
 import {View, Text, StyleSheet} from 'react-native';
-import {MEDIUM_GREY, DARK_GREY, BLACK, LIGHT_GREY} from '../../lib/styles';
+import {MEDIUM_GREY, DARK_GREY, BLACK} from '../../lib/styles';
 import {
   FormattedFieldProp,
   FormattedFieldValue,
 } from '../../sharedComponents/FormattedData';
 import {Field, Observation} from '@comapeo/schema';
+import {ViewStyleProp} from '../../sharedTypes';
 
 export const FieldDetails = ({
   fields,
   observation,
+  style,
 }: {
   fields: Field[];
   observation: Observation;
+  style?: ViewStyleProp;
 }) => {
   return (
     <View>
       {fields.map(field => {
         const value = observation.tags[field.tagKey];
         return (
-          <View
-            key={field.docId}
-            style={[styles.section, styles.optionalSection]}>
+          <View key={field.docId} style={[styles.section, style]}>
             <Text style={styles.fieldTitle}>
```

Comment:
Would it make sense to use the new Text styles here? BodyText? Or HeaderText? The Details (and the regular text) is very light on my screen. It looks different than I think what is asked for in the Figma.
<image src="https://github.com/user-attachments/assets/fdab91d6-588e-4283-b9ee-a41b2a2bc1b7" width="250" />